java/ql/test/experimental/query-tests/security/CWE-522-DecompressionBombs/test.ql

```diff
@@ -0,0 +1,18 @@
+import java
+import experimental.semmle.code.java.security.DecompressionBombQuery
+import TestUtilities.InlineExpectationsTest
+
+module BombTest implements TestSig {
+  string getARelevantTag() { result = "bomb" }
+
+  predicate hasActualResult(Location location, string element, string tag, string value) {
+    tag = "bomb" and
+    exists(DataFlow::Node sink | DecompressionBombsFlow::flowTo(sink) |
+      sink.getLocation() = location and
+      element = sink.toString() and
+      value = ""
+    )
+  }
+}
+
+import MakeTest<BombTest>
```

Comment:
```suggestion
import TestUtilities.InlineFlowTest
import TaintFlowTestArgString<DecompressionBombsConfig, getArgString/2>

string getArgString(DataFlow::Node src, DataFlow::Node sink) {
  exists(src) and
  result = "\"" + sink.toString() + "\""
}
```
You can use `InlineFlowTest` to make test for flow configs more easily. If you just use `TaintFlowTest` (or `DefaultFlowTest`, or `ValueFlowTest`) then it's very concise, but I've also added a string describing the sink, to make it clearer which part of the line is the sink. So please replace `// $bomb` with `// $ hasTaintFlow="zipInputStream"` (or whatever should be between the quotes). Note the space after the `$` - this makes it a bit easier to read, especially when you end up with multiple inline expectations on one line.